fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -216,7 +216,18 @@ def build_xcodebuild_showbuildsettings_command
     def build_settings(key: nil, optional: true)
       unless @build_settings
         command = build_xcodebuild_showbuildsettings_command
-        @build_settings = Helper.backticks(command, print: false)
+        UI.important(command) unless self.xcodebuild_list_silent
```

Comment:
If we have https://github.com/fastlane/fastlane/pull/5188/files?diff=split#r76524918, we can remove this, right?